packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+
+| Property | Type    | Description                                                                               |
+|----------|---------|-------------------------------------------------------------------------------------------|
+| id       | String  | Unique identifier for this Policy (read-only).                                            |
+| name     | String  | Human-readable name for the Policy, configurable during creation or updating.             |
+| type     | String  | Type of the policy. For User Profile Policy objects, this needs to be `Okta:UserProfile`. |
+| status   | String  | `ACTIVE`  or  `INACTIVE`.                                                                 |
+| default  | Boolean | `true` for the first instance of this policy, which gets created by default.              |
+
+### User Profile Policy Object Example
+
+```json
+{
+    "id": "rst10y7ftlPBUWJzu0g4",
+    "name": "Default Policy",
+    "type": "Okta:UserProfile",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y7ftlPBUWJzu0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y7ftlPBUWJzu0g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+
+```
+
+## User Profile Rule Object
+
+| Property    | Type                                                                          | Description                                                                           |
+|-------------|-------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| name        | String                                                                        | Human-readable name for the Rule, configurable during creation or updating.           |
+| id          | String                                                                        | Unique identifier for this Rule (read-only).                                          |
+| type        | String                                                                        | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:SignOn`.  |
+| priority    | Integer                                                                       | Used to determine which Rules take. precedence.                                       |
+| conditions  | Array                                                                         | UserType, User, Group and various Device conditions are supported for this rule type. |
```

Comment:
ditto to sign-on policy rule

Reply:
Same: https://github.com/okta/okta-developer-docs/pull/729/files#r334664386

Reply:
Updated Identifier Match Rule conditions (and conditions example) to be the same as for Sign on Policy Rule.